blast/querier/server/internal/mtx/prometheus.go

```diff
@@ -25,6 +25,8 @@ var (
 			300,  // 5 Minutes
 			600,  // 10 Minutes
 			900,  // 15 Minutes
+			1200, // 20 Minutes
+			1800, // 30 Minutes
```

Comment:
Unrelated change caught in the PR.  Adds to more buckets to the blast command time metrics.